hydra-node/test/Hydra/Chain/Direct/Contract/Mutation.hs

```diff
@@ -127,8 +250,8 @@ propTransactionValidates (tx, lookupUtxo) =
 data SomeMutation = forall lbl.
   (Typeable lbl, Enum lbl, Bounded lbl, Show lbl) =>
   SomeMutation
-  { label :: lbl
-  , mutation :: Mutation
+  { label :: lbl,
+    mutation :: Mutation
   }
```

Comment:
This and other changes in formattiong.. How comes? Different version of `fourmolu`? Not picking up the repository config?